src/main/webapp/script.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Fetches comments from the server and adds them to the DOM. */
+function loadComments() {
+  fetch('/display-comments').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment));
+    })
+  });
+}
+
+/** Fetches results from the server and adds them to the DOM. */
+function loadResults(userQuery) {
+    console.log("Fetching results...");
+    // Capitalize user query because recipe info in Datastore is stored capitalized.
+    fetch('/results?user-query=' + userQuery.toUpperCase()).then(response => response.json()).then((results) => {
+    const resultListElement = document.getElementById('result-list');
+    results.forEach((result) => {
+      console.log("Result found!");
+      resultListElement.appendChild(createResultElement(result));
+    })
+  });
+  return false;
```

Comment:
Why does this function always return false?

Reply:
Because if it doesn't, the page will reload, and results will be lost :( just a nice lil js quirk, apparently; a VSE in cupcake told me about it